documentation/tutorials/get-started-with-phoenix.md

```diff
@@ -0,0 +1,506 @@
+  inputs: ["*.{ex,exs}", "priv/*/seeds.exs", "{config,lib,test}/**/*.{ex,exs}"],
+  subdirectories: ["priv/*/migrations"]
+]
+```
+
+Now in the terminal install these new dependencies.
+
+```bash
+mix deps.get
+```
+
+### Use `AshPostgres.Repo` and Create the Database
+
+Swap `Ecto.Repo` for `AshPostgres.Repo`
+Change your repo module to look like this:
+
+```elixir
+# lib/my_ash_phoenix_app/repo.ex
+
+defmodule MyAshPhoenixApp.Repo do
+  use AshPostgres.Repo, opt_app: :my_ash_phoenix_app
+
+  # installs postgres extensions that ash commonly uses
+  def installed_extensions do
+    ["uuid-ossp", "citext"]
+  end
+end
+```
+
+`AshPostgres.Repo` enriches your repo with additional AshPostgres specific behaviour, but is essentially a thin wrapper around `Ecto.Repo`.
+
+After swapping `AshPostgres.Repo` in you are safe to create the database using:
+
+```
+> mix ash_postgres.create
+
+Compiling 17 files (.ex)
+Generated my_ash_phoenix_app app
+The database for MyAshPhoenixApp.Repo has been created
+```
+
+### Edit Config
+
+We need to specify the Ash API's that our application uses and some config for backwards compatibility that will be removed later.
+Add this to your config:
+
+```elixir
+# config/config.exs
+
+import Config
+
+# For backwards compatibility, the following configuration is required.
+# see https://ash-hq.org/docs/guides/ash/latest/get-started#temporary-config for more details
+config :ash, :use_all_identities_in_manage_relationship?, false
+
+config :my_ash_phoenix_app,
+  ash_apis: [MyAshPhoenixApp.Blog]
+```
+
+### Create API
+
+You may have noticed that `MyAshPhoenixApp.Blog` doesn't actually exist yet. Lets make it now.
+
+```elixir
+# lib/my_ash_phoenix_app/blog.ex
+
+defmodule MyAshPhoenixApp.Blog do
+  use Ash.Api
+
+  resources do
+    registry MyAshPhoenixApp.Blog.Registry
+  end
+end
+```
+
+This tiny module tells Ash that `MyAshPhoenixApp.Blog` is an Ash API. It can be thought of as a [Bounded Context](https://martinfowler.com/bliki/BoundedContext.html) in Domain Driven Design terms and can seen as analogous to a Phoenix context.
+
+It also tells where to find the resources within this API - in the `MyAshPhoenixApp.Blog.Registry` which we will make now.
+
+### Create Registry
+
+```elixir
+# lib/my_ash_phoenix_app/blog/registry.ex
+
+defmodule MyAshPhoenixApp.Blog.Registry do
+  use Ash.Registry,
+    extensions: [
+      # This extension adds helpful compile time validations
+      Ash.Registry.ResourceValidations
+    ]
+
+  entries do
+    entry MyAshPhoenixApp.Blog.Post
+  end
+end
+```
+
+In the registry we enumerate the Ash resources we want our Ash API to contain. In this case its just one resource called `Post`.
+
+### Creating the `Post` Resource
+
+Its convention to place all the resource in their own resources folder. So when we create `Post` we will place it in `lib/my_ash_phoenix_project/blog/resources/post.ex`. So the structure after making the resource should look like so:
```

Comment:
`It's convention`

Reply:
addressed: [5c2cb7f](https://github.com/ash-project/ash/pull/504/commits/5c2cb7fdf6732e6b12681b9cd23819322f0baa87)